src/main/java/org/broadinstitute/hellbender/tools/pon/coverage/pca/HDF5PCACoveragePoN.java

```diff
@@ -0,0 +1,483 @@
+package org.broadinstitute.hellbender.tools.pon.coverage.pca;
+
+import com.google.common.collect.ImmutableMap;
+import htsjdk.samtools.util.Lazy;
+import org.apache.commons.math3.linear.RealMatrix;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.tools.exome.ReadCountCollection;
+import org.broadinstitute.hellbender.tools.exome.Target;
+import org.broadinstitute.hellbender.tools.exome.TargetTableColumn;
+import org.broadinstitute.hellbender.tools.pon.allelic.AllelicPanelOfNormals;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.hdf5.HDF5File;
+
+import java.io.File;
+import java.util.*;
+import java.util.stream.Collectors;
+
+/**
+ * HDF5 File backed coverage panel of normals data structure.
+ *
+ * Currently, we assume the coverage panel of normals is always fully specified but that the allelic panel of normals is
+ * essentially optional.
```

Comment:
oops, forgot to remove this comment, thanks for catching it!  (originally i tried to add the aPoN as an optional field of the coverage PoN, but scrapped that for various reasons)